ores/score_caches/empty.py

```diff
@@ -8,3 +8,9 @@ def lookup(self, *args, **kwargs):
 
     def store(self, *args, **kwargs):
         return None
+
+    @classmethod
+    def from_config(cls, config, name, section_key="score_caches"):
+        section = config[section_key][name]
```

Comment:
Flake8 error: ores/score_caches/empty.py:14:9: F841 local variable 'section' is assigned to but never used

Reply:
Thanks!  Got it.